### ISSUE TITLE
Custom Blips Don't Update

### ISSUE THREAD

**Issue by JamesEU** [citizenfx/fivem#951]
Hi, so when you have custom blips that are streamed using `minimap.ytd` they do not get 'redownloaded' when updated! So users then keep the old ytd and not the updated ones! The only "resolution" for this is to clear the cache for that server which then resolves the issue. 

Repro Steps: 

1. Get a minimap.ytd with the texture blips_texturesheet_ng being replaced with 1 icon for example replacing the dollar sign next to the T on this specific texture!
2. Create the blip using AddBlipForCoord and set its sprite to 207 if you are using the icon specified above. 
3. Go into your localhost/server and see that the blip is updated to the one icon you have set. 
4. Then leave your server and change the icon again to something else and replace the existing ytd in your stream folder. (Shutting down the server or keeping it open make no difference to the result the same issue will apply either way)
5. Then once you have done that rejoin back and you'll see that the blip is still the same as the old one! This will still be the same until you clear the server cache for that server on your client which will then redownload the new blip and upon logging in you'll see it has been updated

**Comment by blattersturm** [comment]
Have you _restarted_ the resource on the server? That step seems.. missing.

**Comment by JamesEU** [comment]
Yeah, issue #859 sounds like the same relative issue I will test the restarted

Have I restarted the resource: Yes, same issue. 

I'm using OpenIV to save the modified file and once I save it I am copying it to my streamed file. I have double-checked that I am streaming the updated file by re-adding it back into OpenIv and seeing that it's not the same as the original blip I had there also the size of the files is different also.

**Comment by blattersturm** [comment]
>leave your server

Wait, did you mean _by closing the game_ or by _using the disconnect option_? Are you _sure_ that it's not magically fine when just restarting the game?

**Comment by JamesEU** [comment]
By restarting I mean disconnecting and then rejoining and also the issue
persists even if you quit and then rejoin.

**Comment by blattersturm** [comment]
Can you note down some _conclusive_ steps for this that don't look like they're written from memory? I couldn't replicate this so far.

**Comment by JamesEU** [comment]
Okay, 

1. Get a minimap ytd attached below: 

[The one with the good pound sign ](https://mega.nz/file/88wUHD5B#Jw22YEgvxDTA2VbBvAy56hkDfpftSjBM8gK1cGyAvVw)

[The one with the bad pound sign](https://mega.nz/file/1pxQmL7a#sTu8OGnU9ouQh31zHKifIFLpGjTBcZsbDRyHnQAGcoI)

2. Stream the assets in a resource in the /stream
Use the good pound sign file first and draw the blip. In this case, the sprite id is 207.

3.  Go into the game with the resource already started and blip already drawn.

4. You'll see the good pound sign being this image

![image](https://user-images.githubusercontent.com/60899490/132143162-daf4b776-2f5f-4301-9039-ea67a540641f.png)

5. Then replace the minimap.ytd with the "bad pound sign" and then either restart the resource, leave the server by disconnecting, or by closing FiveM completely using the 'quit command' all will reproduce the same result.  

What is expected to happen is it'll update with this symbol: 

![image](https://user-images.githubusercontent.com/60899490/132143245-a89c9857-6e24-40b5-b4f6-e7040d614743.png)

However, this is not the case I have been able to reproduce this issue with friends also on multiple occasions now.

**Comment by blattersturm** [comment]
Yeah, no, I can't replicate this.

What I did:
1. Made a resource named `blipp`, dropped in the first minimap.ytd file into stream/.
2. Started the server + resource and joined it.
3. Observed the blip being the pink one.
4. Quit the game.
5. Replaced the minimap.ytd with the second one, and restarted the resource.
6. Started the game and joined the server.
7. Observed the blip being correctly black,
